lib/mogrify.ex

```diff
@@ -249,6 +252,16 @@ defmodule Mogrify do
     |> put_frame_count(output)
   end
 
+  @doc """
+  Provides "identify" information about an image with raw access attribute.
+  Example: identify(file_path, "'%[orientation]'")
+  """
+  def identify(file_path, option) do
```

Comment:
I don't think this API will be appropriate. ImageMagick's `identify` command can take many options, not just `-format`. https://imagemagick.org/script/identify.php

Maybe better to pass in a Keyword list of options:

    identify(path, format: "'%[orientation]'")

And then check for the `format` option and handle that specially. This way, future options can be added.

(If no handled options are present in the Keyword list, it falls back to `identify/1` code path.)